src/lib/holocene/api-pagination.svelte

```diff
@@ -29,11 +31,14 @@
     itemsKeyname?: string;
     previousButtonLabel: string;
     nextButtonLabel: string;
+    filterable?: boolean | undefined;
```

Comment:
We should replace the [custom input](https://github.com/temporalio/ui/blob/main/src/lib/pages/schedules.svelte#L84) on the Schedules page ⬇️  with this prop 🎉 

![Screenshot 2024-06-03 at 11 21 50 AM](https://github.com/temporalio/ui/assets/15069288/109b7362-6d54-4a7c-927d-a539206e6b3d)

Reply:
We should, I can make a follow up task for that